docs/users/overview.adoc

```diff
@@ -9,10 +9,11 @@ You will quickly have an overview of the `bpmn-visualization` capabilities.
 
 === Usage
 
-`bpmn-visualization` can be directly used in HTML pages or integrated in Javascript and TypeScript applications. It is distributed as a https://npmjs.org/package/bpmn-visualization:[npm package]
+`bpmn-visualization` can be directly used in HTML pages or integrated in Javascript and TypeScript applications. It is distributed as a https://npmjs.org/package/bpmn-visualization[npm package]
 in various bundle formats.
 
-The API documentation is provided directly in the package thanks to the TypeScript declaration files that can used in IDE for code assist.
+The API documentation is provided directly in the package thanks to the TypeScript declaration files that can be used in IDE for code assist.
+There is also a html version of API which can be previewed https://process-analytics.github.io/bpmn-visualization-js/api/index.html[here, window="_blank"].
```

Comment:
❓ why not using a relative link to make it work locally or on whatever environment?

Reply:
yes I kept the docs so we can run both at the time
I however forgot to change the call in the workflow - done